src/hacking/testing.md

```diff
@@ -192,6 +199,12 @@ Once the log is saved, run from the root directory:
 
     ./mach update-wpt /tmp/servo.log
 
+Running all Web Platform Tests locally will take a long time and usually cause unrelated failures (such as the runner exceeding the maximum number of open files on your system).
```

Comment:
Might be worth mentioning that current expectations are set for CI machines (another reason for unrelated failures).